doc/conf.py

```diff
@@ -260,7 +261,10 @@
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-# intersphinx_mapping = {'http://docs.python.org/': None}
+intersphinx_mapping = {
+    'http://docs.python.org/': None,
+    'https://docs.djangoproject.com/en/2.2': 'https://docs.djangoproject.com/en/2.2/_objects',
```

Comment:
Why not "stable" instead of "2.2"?

Reply:
See my comments from the other PR, you're reviewing in the wrong order :-D